docs/content/docs/user-docs/irsa.md

```diff
@@ -63,7 +72,10 @@ For detailed instructions, refer to Amazon EKS documentation on how to [create a
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
 OIDC_PROVIDER=$(aws eks describe-cluster --name $EKS_CLUSTER_NAME --region $AWS_REGION --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
 ACK_K8S_NAMESPACE=ack-system
-ACK_K8S_SERVICE_ACCOUNT_NAME=ack-s3-controller
+
+# Update the service name variables as needed
+AWS_SERVICE_NAME="s3"
+ACK_K8S_SERVICE_ACCOUNT_NAME=ack-$AWS_SERVICE_NAME-controller
```

Comment:
nit: Move to top of the code block. 
nit: Rename `AWS_SERVICE_NAME` to `SERVICE` (same as `install.md`) - other code blocks in this document will need to be updated to match.